src/Preference.h

```diff
@@ -8,11 +8,27 @@
 #include "RageUtil.h"
 class XNode;
 
+enum class PreferenceFlag

```

Comment:
Slight nit, but can we call it `PreferenceType`?

Reply:
My initial design was different, but the name stuck. Changed.